frontend/ui/widget/multiinputdialog.lua

```diff
@@ -1,3 +1,80 @@
+--[[--
+Widget for taking multiple user inputs.
+
+Example for input of two strings and a number:
+
+    local MultiInputDialog = require("ui/widget/multiinputdialog")
+    local @{ui.uimanager|UIManager} = require("ui/uimanager")
+    local @{gettext|_} = require("gettext")
+
+    local sample_input
+    sample_input = MultiInputDialog:new{
+        title = _("Title to show"),
+        fields = {
+            {
+                description = _("Describe this field"),
+                input_type = "string",
```

Comment:
I don't know why there are so many lines like this one in the code.
Setting `input_type` to "string" or "text" is absolutely unnecessary in input dialogs.

Reply:
Well that is a good point. Maybe because the example in http://koreader.rocks/doc/modules/ui.widget.inputdialog.html leads people to code it?

I would suggest, that we comment the line in the doc to something like 
`--input_type="text"  -- this is default` so that it is documented that there is at least another type possible.

Reply:
Kind of instruction
https://github.com/koreader/koreader/blob/6d40c7c7ea8abe2e12e9897ee7d6ed0dbc5c8712/frontend/ui/widget/inputtext.lua#L26